seed/static/seed/js/services/geocode_service.js

```diff
@@ -29,7 +29,7 @@ angular.module('BE.seed.service.geocode', [])
 
       geocode_factory.check_org_has_api_key = function (org_id) {
         var params = {organization_id: org_id};
-        return $http.get('/api/v2/geocode/api_key_exists', {
+        return $http.get('/api/v3/organizations/' + org_id + '/api_key_exists', {
```

Comment:
Getting a 404 with this service (tested loading the mapping page). I think we need to change this to `'/api/v3/organizations/' + org_id + '/geocode_api_key_exists/'`

Reply:
So regarding this...I think I just changed all the "v2" in this .js file where in this case I wasn't working directly on check_org_has_api_key. Would you rather me change it back to v2 in case someone else is working the ticket related to this call?